### ISSUE TITLE
Add issue template for bugs, features etc.

### ISSUE THREAD

**Issue by melihyilmaz** [Noble-Lab/casanovo#32]
This can wait until we have release notes and versions for Casanovo

**Comment by bittremieux** [comment]
Considering the number of issues with incomplete information that we've been receiving recently, it might be worth it to prioritize creating an issue template that request detailed relevant information (full Casanovo log, OS properties, input files used, proper problem description).

**Comment by Lilferrit** [comment]
I've added a bug report template in the [bug-report-template](https://github.com/Noble-Lab/casanovo/tree/bug-report-template) branch under the `.github` directory. I implemented the template as a markdown template, however it looks like alternatively it could be defined as a yaml file, which would be used to generate a web form users can fill out instead of a markdown template. This may be more user friendly, but that specific GitHub feature is still in beta atm.

Let me know if I need to add anything to the template or make any other changes.
